Sources/MapboxCoreNavigation/Router.swift

```diff
@@ -167,6 +167,15 @@ public protocol Router: CLLocationManagerDelegate {
     func updateRoute(with indexedRouteResponse: IndexedRouteResponse,
                      routeOptions: RouteOptions?,
                      completion: ((Bool) -> Void)?)
+
+    /**
+     Asynchronously removes currently active route.
+
+     This method is called automatically by `NavigationViewController` at the end of the navigation.
+     If you use `Router` without `NavigationViewController`, you are supposed to call this method when you are done
```

Comment:
It would be great to add this to the CHANGELOG. As it was said, it's not a breaking change, but something we should proactively point developers to.